### ISSUE TITLE
clickhouse_driver.errors.UnknownPacketFromServerError: Code: 100. Unknown packet XXX from server

### ISSUE THREAD

**Issue by wawaka** [mymarilyn/clickhouse-driver#69]
This error happens for some queries only.
```Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/clickhouse_driver/client.py", line 137, in execute
    columnar=columnar
  File "/usr/local/lib/python3.7/site-packages/clickhouse_driver/client.py", line 208, in process_ordinary_query
    columnar=columnar)
  File "/usr/local/lib/python3.7/site-packages/clickhouse_driver/client.py", line 50, in receive_result
    return result.get_result()
  File "/usr/local/lib/python3.7/site-packages/clickhouse_driver/result.py", line 39, in get_result
    for packet in self.packet_generator:
  File "/usr/local/lib/python3.7/site-packages/clickhouse_driver/client.py", line 62, in packet_generator
    packet = self.receive_packet()
  File "/usr/local/lib/python3.7/site-packages/clickhouse_driver/client.py", line 76, in receive_packet
    packet = self.connection.receive_packet()
  File "/usr/local/lib/python3.7/site-packages/clickhouse_driver/connection.py", line 352, in receive_packet
    packet_type, self.get_description()
clickhouse_driver.errors.UnknownPacketFromServerError: Code: 100. Unknown packet 109 from server <HOST>:<PORT>
```
It was introduced in commit d47a263df27221cfc35bea23e6a7f18dd9e9a665

**Comment by xzkostyan** [comment]
Hi!

Can you provide minimal data and query to reproduce this issue?

**Comment by xzkostyan** [comment]
@wawaka check this PR. Does your error still occur on latest version?

**Comment by xzkostyan** [comment]
@aido93 Yes, ClickHouse native protocol can work with only one query per connection. I'll add note about this to docs.

**Comment by xzkostyan** [comment]
It seems that the issue was resolved by #72.

**Comment by wawaka** [comment]
I confirm, it is fixed since 0.0.18
